features/reply_to_message.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Reply to message
+    As a user
+    In order to send a response
+    I want to be able to reply to a message in my inbox
+
+Scenario: Submit sign up form [Happy path]
+- (Given) some context
+- (When) some action is carried out
+- (Then) a particular set of observable consequences should obtain
```

Comment:
Follow the standard here `-(Given)` is not needed just do `Given`

Reply:
removed